README.md

```diff
@@ -1 +1,171 @@
-# newyork_restaurant_grading
\ No newline at end of file
+# Prediction for the Grading of New York City Restaurant
+
+  - authors (ordered alphabetically by last name) from Group 18:
+    Nikita Susan Easow, Sneha Sunil, Edward (Yukun) Zhang, Lauren Zung
+  
+
+A data analysis project for DSCI 522 (Data Science workflows); a
+course in the Master of Data Science program at the University of
+British Columbia.
+
+URL of the project repo: https://github.com/UBC-MDS/newyork_restaurant_grading.git
+
+## Project Proposal
+
+We choose the large dataset DOHMH New York City Restaurant Inspection Results sourced from 
+NYC OpenData Portal. It is retrieved from the tidytuesday repository by Thomas Mock, 
+and can be sourced [here](https://github.com/rfordatascience/tidytuesday/tree/master/data/2018/2018-12-1.).
+The original data set can be found [here](https://data.cityofnewyork.us/Health/DOHMH-New-York-City-Restaurant-Inspection-Results/43nn-pn8j/data).
+It contains the violation citations from every inspection conducted for restaurants in New York City from 2012 to 2018. 
+Each row represents a restaurant that has been assessed by a health inspector, 
+including information about their business such as the restaurant name, phone number, location (borough, building number, street, zip code) 
+and type of cuisine, as well as the details about their inspection 
+(e.g. date, violation code, description, whether there were any violations cited, whether they were critical, etc.). 
+The restaurants can be assigned an official grade of A, B, or C, otherwise they are assigned Z or P for pending review.
+
+As data scientists, we are interested in predicting the restaurant grading in New York City
```

Comment:
For this part (showing one main prediction question), I think we could also include multiple sub questions (as mentioned in the milestone doc) like the following:
1. Which cuisines is more likely to be graded A in New York?
2. Which cuisine is more likely to be graded B in New York?
3. Which borough in New York seems to have the best restaurants?
4. Which borough in New York seem to have the most restaurants with the most code violations?

We could probably come up with more than these as well - what do you think?